scripts/cli.sh

```diff
@@ -0,0 +1,107 @@
+#! /usr/bin/env bash
+
```

Comment:
I think we should add a comment here to explain that this script should not be run directly, but only ever through `make cli`

```suggestion
# IMPORTANT
# This script should NOT be run directly.
# Run `make cli` from the root of the repository instead.

```